### PR TITLE
fix broadcasting bug in fitters

### PR DESCRIPTION
**Pull request by nden**
Fixes the broadcasting bug in #5737.

### CODE REVIEW

File: astropy/modeling/tests/test_fitters.py [hunk 1]
```diff
@@ -617,3 +617,24 @@ def test_with_fitters_and_sigma_clip(self):
         _, fitted_model = fit(g2_init, self.x, self.y, self.z)
         assert_allclose(fitted_model.parameters[0:5], self.model_params,
                         atol=1e-1)
+
+def test_fitters_with_weights():
```
Comment (mirca) [inline astropy/modeling/tests/test_fitters.py]: Looks like it needs ``@pytest.mark.skipif('not HAS_SCIPY')``

File: CHANGES.rst [hunk 2]
```diff
@@ -225,6 +225,9 @@ Bug Fixes
 
   - Fixed a problem with setting ``bounding_box`` on 1D models. [#5718]
 
+  - Fixed a broadcasting problem with weighted fitting of 2D models
+    with ``LevMarLSQFitter``. [#5737]
```
Comment (bsipocz) [inline CHANGES.rst]: @astrofrog - apparently astrobot was sleeping when this was merged, I'm opening an issue and will try to fix it later.

**Comment by astrofrog** [comment]
@bsipocz - I changed the name of the heroku app the other day and forgot to change it in the astropy settings... anyway I redelivered the payload to test if this PR was correctly flagged, and it all works now.

**Comment by bsipocz** [comment]
@astrofrog - Thanks. Mystery of the day is solved then :)

**Comment by nden** [comment]
Of course I did this on purpose, to point out the problem :-). But seriously @bsipocz , how did you spot this?

**Comment by bsipocz** [comment]
@nden - I was looking at how we stand in terms of 1.3.1, and the pre-releasing scripts (https://github.com/astropy/astropy-tools/tree/master/pr_consistency) made a nice big red warning about it.

File: astropy/modeling/tests/test_fitters.py [hunk 3]
```diff
@@ -617,3 +617,26 @@ def test_with_fitters_and_sigma_clip(self):
...
+    Xin, Yin = np.mgrid[0:21, 0:21]
+    fitter = LevMarLSQFitter()
+
+    with NumpyRNGContext(_RANDOM_SEED):
+        zsig = np.random.normal(0, 0.01, size=Xin.shape)
+
+    # Non-linear model
+    g2 = models.Gaussian2D(10, 10, 9, 2, 3)
+    z = g2(Xin, Yin)
+    gmod = fitter(models.Gaussian2D(15, 7, 8, 1.3, 1.2), Xin, Yin, z + zsig)
```
Comment (majkelx) [inline astropy/modeling/tests/test_fitters.py]: Contrary to name `test_fitters_with_weights`, there are no weights used, if I have not overlooked anything...

**Pull request merged**